libheif/plugins/encoder_svt.cc

```diff
@@ -674,7 +674,7 @@ struct heif_error svt_encode_image(void* encoder_raw, const struct heif_image* i
   svt_config.logical_processors = encoder->threads;
 
   // disable 2-pass
-  svt_config.rc_stats_buffer = (SvtAv1FixedBuf) {nullptr, 0};
+  svt_config.rc_stats_buffer = {nullptr, 0};
```

Comment:
@fishmonger42 This part already got applied (see https://github.com/strukturag/libheif/pull/1059) and is probably the cause of the merge conflicts. Possibly you just need to drop it.